insight.py

```diff
@@ -126,6 +127,22 @@ def run_perf(self, args):
             self.insight_perf = perf.InsightPerf(options=args)
         self.insight_perf.run(self.full_outdir)
 
+    def run_ftrace(self, args):
+        if not args.ftrace:
+            logging.debug("Ignoring collecting of ftrace data.")
+            return
+        # perf requires root priviledge
+        if not util.is_root_privilege():
+            logging.fatal("It's required to run ftrace with root priviledge.")
+            return
+
+        if args.ftracepoint:
+            self.insight_ftrace = ftrace.InsightFtrace(vars(args))
```

Comment:
Would it be better to `var()` in `InsightFtrace`'s `__init__()`?

Reply:
I agree with you.